### PR TITLE
Feature/dropdown v2

### PR DESCRIPTION
**Pull request by panvourtsis**
## Description
This PR implements the designs for the simple dropdown. 

This dropdown will be able to get options and show them. Each option must have a specific shape. The component can take a default value from the options in order to start with that. Also each select is filterable.

React-Select library also has been removed as it makes things complicated.

Multiple dropdown will be splitted to another PR as the checkbox that will use is under some minor fixes

depends on #165 

## Screenshot
image
image

## Test Plan
- storyshots
- design qa

### CODE REVIEW

File: src/components/DatePicker/DatePickInput/DatePickInput.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import dayjs from 'dayjs';
 import * as React from 'react';
+import omit from 'lodash/omit';
 import { DayPickerInputProps } from 'react-day-picker';
 import useTheme from '../../../hooks/useTheme';
```
Comment (gsmetak) [inline src/components/DatePicker/DatePickInput/DatePickInput.tsx:5]: Is a blank line required above this one, to separate the imports?

**Review by gsmetak — APPROVED** [review]

File: src/components/DatePicker/DatePickInput/DatePickInput.tsx (hunk 1 above)
Comment (panvourtsis) [reply src/components/DatePicker/DatePickInput/DatePickInput.tsx:5]: not quite sure what do you mean here? can you send an example?

Comment (kouloumos) [reply src/components/DatePicker/DatePickInput/DatePickInput.tsx:5]: maybe he is referring to a styling that separates imports from external libraries, internal etc?

Comment (panvourtsis) [reply src/components/DatePicker/DatePickInput/DatePickInput.tsx:5]: actually i have such thing in my notes to implement to all projects. But as doing this manually is actually us shooting at our feet i will import a solution to do this with a rule

**Pull request merged**